src/hacbs/utils/commits-utils.ts

```diff
@@ -0,0 +1,62 @@
+import { PipelineRunKind, Commit } from '../types';
+
+export const COMMIT_ANNOTATION = 'build.appstudio.redhat.com/commit_sha';
+export const COMMIT_URL_ANNOTATION = 'build.appstudio.redhat.com/sha_url';
+export const COMMIT_BRANCH_ANNOTATION = 'build.appstudio.redhat.com/target_branch';
+export const COMMIT_COMPONENT_LABEL = 'build.appstudio.redhat.com/component';
+export const COMMIT_USER_LABEL = 'build.appstudio.redhat.com/user';
+export const COMMIT_REPO_URL = 'build.appstudio.redhat.com/sha_url';
```

Comment:
move these consts to `PipelineRunLabel` enum in `src/hacbs/consts/pipelinerun.ts` file